netcore/Makefile

```diff
@@ -47,6 +51,16 @@ TEST_ASSETS_URL = $(FEED_BASE_URL)/corefx-tests/$(NETCORETESTS_VERSION)/$(TESTS_
 BLAME = $(shell git blame ../configure.ac | grep AC_INIT | cut -f1 -d' ')
 VERSTUB = .$(shell git rev-list --count $(BLAME)..HEAD)
 
+all: runtime bcl
+
+../.dotnet/dotnet:
+	bash ../eng/common/dotnet-install.sh -v $(NETCOREAPP_VERSION)
+	bash ../.dotnet/dotnet-install.sh  -c master -i ../.dotnet
```

Comment:
I am not sure this is the right way doing it, it'll install "random" master version. Shouldn't DARC sync take care of this dependency?

@ViktorHofer any inputs on that?

Reply:
As discussed with @steveisok offline, you should not clal into `eng/common/dotnet-install.sh` but into the InitializeToolset function in tools.sh: `. ../eng/common/tools.sh && InitializeToolset`. This should install the version in the global.json file.

Reply:
@ViktorHofer, I ran into some additional issues I'd like @EgorBo to help validate.  Depending on the outcome, we can discuss at our weekly meeting.

Reply:
@ViktorHofer  I am trying to call that function but getting:
```
/prj/mono/mono5/artifacts/toolset/restore.proj : error : Unable to find package Microsoft.DotNet.Arcade.Sdk with version (= 1.0.0-beta.19274.6)
/prj/mono/mono5/artifacts/toolset/restore.proj : error :   - Found 1422 version(s) in dotnet-core [ Nearest version: 1.0.0-dev.18424.13 ]
/prj/mono/mono5/artifacts/toolset/restore.proj : error :   - Found 0 version(s) in nuget.org
/prj/mono/mono5/artifacts/toolset/restore.proj : error MSB4236: The SDK 'Microsoft.DotNet.Arcade.Sdk' specified could not be found.
```
is there an additional nuget feed I should add to nuget.config?

Reply:
> UPD updated version to 1.0.0-dev.18424.13 in the global.json

 ?

Reply:
If `1.0.0-beta.19274.6` doesn't work, try it again with `1.0.0-beta.19278.1`. Weird...

Reply:
@ViktorHofer both don't work for me, I have no idea why, I do see these versions in my Nuget Explorer

Reply:
@ViktorHofer ah, `dotnet nuget locals all --clear` helped

Reply:
To quickly validate if a package exists in a feed you can use https://dotnetfeed.blob.core.windows.net/dotnet-core/sleet.packageindex.json